core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.ts

```diff
@@ -433,6 +522,10 @@ angular.module('oppia').component('settingsTab', {
         ctrl.canReleaseOwnership = false;
         ctrl.canUnpublish = false;
         ctrl.explorationId = ExplorationDataService.explorationId;
+        ctrl.loggedinUser = null;
```

Comment:
```suggestion
        ctrl.loggedInUser = null;
```
ditto everywhere this is used.

Reply:
Done

Reply:
Done